### PR TITLE
24811 - Remove transaction counts from transaction table

### PR DESCRIPTION
**Pull request by seeker25**
*Issue #:*
https://github.com/bcgov/entity/issues/24811

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/composables/transactions-factory.ts [hunk 1]
```diff
@@ -74,8 +74,7 @@ export const useTransactions = () => {
         currentOrganization.value.id, transactions.filters, viewAll.value)
       if (response?.data) {
         transactions.results = response.data.items || []
-        transactions.totalResults = response.data.total
-
+        transactions.totalResults = transactions.results.length * response.data.page + (response.data.hasMore ? 1 : 0)
```
Comment (seeker25) [inline auth-web/src/composables/transactions-factory.ts:77]: This controls the next page button

File: auth-web/src/composables/transactions-factory.ts [hunk 2]
```diff
@@ -94,7 +93,7 @@ export const useTransactions = () => {
         })
         if (transactions.results.some((transaction: Transaction) => transaction.refundDate)) {
           transactions.results.sort((transaction1: Transaction, transaction2: Transaction) => {
-            return new Date(transaction2.createdOn).getTime() - new Date(transaction1.createdOn).getTime()
+            return moment(transaction2.createdOn).valueOf() - moment(transaction1.createdOn).valueOf()
```
Comment (seeker25) [inline auth-web/src/composables/transactions-factory.ts:96]: new Date() -> using moment instead, incase new Date() is in a different TZ

File: auth-web/src/services/payment.services.ts [hunk 3]
```diff
@@ -122,7 +122,7 @@ export default class PaymentService {
     if (viewAll) params.append('viewAll', `${viewAll}`)
 
     const url = `${ConfigHelper.getPayAPIURL()}/accounts/${accountId}/payments/queries`
-    return axios.post(url, filterParams.filterPayload, { params })
+    return axios.post(url, { ...filterParams.filterPayload, excludeCounts: true }, { params })
```
Comment (seeker25) [inline auth-web/src/services/payment.services.ts:125]: always using this param

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-3178-mhfsvf2l.web.app

**Review by ochiu — APPROVED** [review]

**Pull request merged**